cmd/main.go

```diff
@@ -98,7 +98,9 @@ func findCurrentRepo() (string, error) {
 		// give up, let the user figure it out
 		return "", fmt.Errorf("could not determine repository path from module data, package data, or by initializing a module: %v", err)
 	}
-	defer os.Remove("go.mod") // clean up after ourselves
+	defer func() {
+		_ = os.Remove("go.mod")
+	}() // clean up after ourselves
```

Comment:
What is the motivation of this change?

Reply:
Same as with the previous change, the error is explicitly handled by using `_`. The way to do so is using a wrapper funtion.

Reply:
The same as above. Note that `Command clean up` is not leading a good understanding of your motivation and what and why is done here. Also, I'd recommend you look for the golang good practices and check if it is actually recommended for it in order to have a good argumentation/justification for this change. It is important to highlight that in POV the changes could not be done in just a single place in this scenario and the same standard should be applied across all repo.